src/unix/index_unix.ml

```diff
@@ -26,17 +26,17 @@ let current_version = "00000001"
 module Stats = Index.Stats
 
 module IO : Index.IO = struct
-  external ( ++ ) : int64 -> int64 -> int64 = "%int64_add"
+  let ( ++ ) = Int63.add
```

Comment:
Could (and perhaps should) inline this again using `Int63.observe`.

Reply:
On second thought, this isn't inline-able anyway since the 32-bit representation isn't semantically `int64`.